### ISSUE TITLE
Error add custom transcoding profile to live stream

### ISSUE THREAD

**Issue by Izorkin** [Chocobozzz/PeerTube#3992]
**Describe the current behavior**
Error add custom transcoding profile to live stream

**Steps to reproduce:**

1. Install plugin `transcoding-profile-debug`
2. Add transcoding profiles:
```
{
  "vod": [],
  "live": [
    {
      "encoderName": "aac",
      "profileName": "custom-live",
      "outputOptions": []
    },
    {
      "encoderName": "libx264",
      "profileName": "custom-live",
      "outputOptions": []
    }
  ]
}
```
and encoders priorities
```
{
  "vod": [],
  "live": [
    {
      "encoderName": "aac",
      "streamType": "audio",
      "priority": 1000
    },
   {
      "encoderName": "libx264",
      "streamType": "video",
      "priority": 1000
    }
  ]
}
```

**Additional information**
* Debug log:
```
npm[124734]: [custom_peertube] 2021-04-19 23:17:46.106 debug: Available transcoding profiles built. {
npm[124734]:   "availableProfiles": {
npm[124734]:     "vod": [
npm[124734]:       "default",
npm[124734]:       "custom-live"
npm[124734]:     ],
npm[124734]:     "live": [
npm[124734]:       "default"
npm[124734]:     ]
npm[124734]:   }
npm[124734]: }
npm[124734]: [custom_peertube] 2021-04-19 23:17:46.106 debug: Available transcoding profiles built. {
npm[124734]:   "availableProfiles": {
npm[124734]:     "vod": [
npm[124734]:       "default",
npm[124734]:       "custom-live"
npm[124734]:     ],
npm[124734]:     "live": [
npm[124734]:       "default"
npm[124734]:     ]
npm[124734]:   }
npm[124734]: }
```

* PeerTube instance:
  * version: 3.0.1
  * NodeJS version: 14.16.1
  * Ffmpeg version: 4.3.2

**Comment by Chocobozzz** [comment]
Thanks, it's a bug in the plugin. I fixed it in 0.0.2

**Comment by Izorkin** [comment]
With this configuration:
```
"live": true
  "enabled": true,
  "transcoding": {
    "enabled": false,
    "profile": "custom-live"
  }
```
After the end of the live translation, the "default" profile is used to compress the video.
How to apply a "custom-live" profile without activating transcoding in online mode?
